README.md

```diff
@@ -104,9 +105,36 @@ curl -X GET https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=foo
 
 Google Invalid Token Response
 ```JavaScript
-curl -X GET https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=foo 
+curl -X GET https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=foo
 400 Bad Request
 {
  "error_description": "Invalid Value"
 }
-```
\ No newline at end of file
+```
+
+[Login with Amazon](https://login.amazon.com/website)
+
```

Comment:
Clarify here that the token response from the first call is used to obtain the profile object in the second call. Clarify also why two calls are needed.